code/ZAS/Fire.dm

```diff
@@ -7,6 +7,9 @@ If it gains pressure too slowly, it may leak or just rupture instead of explodin
 */
 
 //#define FIREDBG
+#define FIRE_LIGHT_1	2 //These defines are the power of the light given off by fire at various stages
```

Comment:
#undef at the end of file if not used outside (and ctrl-F tells me they're not).